scale/job/configuration/interface/error_interface.py

```diff
@@ -92,10 +92,6 @@ def get_error(self, exit_code=None):
                     # get the name to search for in the 'Error' database table
                     error = self._lookup_error(error_name)
 
-        # Set the error to the 'Unknown' error as a fall back default
-        if not error:
-            if exit_code is None or exit_code != 0:
-                error = Error.objects.get_unknown_error()
         return error
```

Comment:
The documentation for this method should indicate the possibility of a None return value